test/client.spec.ts

```diff
@@ -138,51 +138,51 @@ describe(OpenFeatureClient.name, () => {
     });
 
     describe('detailed flag evaluation', () => {
-      describe(` ${OpenFeatureClient.prototype.getBooleanDetails.name}`, () => {
+      describe('getBooleanDetails', () => {
         it('should return boolean details, and call boolean resolver', async () => {
           const booleanFlag = 'my-boolean-flag';
           const defaultBooleanValue = false;
           const booleanDetails = await client.getBooleanDetails(booleanFlag, defaultBooleanValue);
 
           expect(booleanDetails.value).toEqual(BOOLEAN_VALUE);
           expect(booleanDetails.variant).toEqual(BOOLEAN_VARIANT);
-          expect(MOCK_PROVIDER.resolveBooleanEvaluation).toHaveBeenCalledWith(booleanFlag, defaultBooleanValue, {});
+          expect(MOCK_PROVIDER.resolveBooleanEvaluation).toHaveBeenCalledWith(booleanFlag, defaultBooleanValue, {}, {});
         });
       });
 
-      describe(OpenFeatureClient.prototype.getStringDetails.name, () => {
```

Comment:
This makes me sad. These tests are all brittle now.

I'm not really a fan of changing these just for particular tooling.